test/mailers/admin_mailer_test.rb

```diff
@@ -0,0 +1,44 @@
+require 'test_helper'
+
+class AdminMailerTest < ActionMailer::TestCase
+  def setup
+    @event = make_event
+    @email = 'test@admin.com'
```

Comment:
Maybe we should name this instance variable differently as is the target email address and in the test we are also creating email (I got confused when reading the test again)

Reply:
@admin = make_admin

and later use @admin.email